### PR TITLE
Support Rails >= 6.0 and Ruby >= 3

### PR DESCRIPTION
**Pull request by bivanalhar**
What is tested to be compatible so far:
- Rails 6.0, 6.1, 7.0, 7.1
- Ruby 3.0, 3.1

### CODE REVIEW

File: README.md
Comment (cysjonathan) [inline README.md:1]: You should turn off prettier onFileSave to prevent so many cosmetic changes to the file.

File: Appraisals [hunk 1]
```diff
@@ -1,15 +1,19 @@
 appraise "rails-6.0" do
   gem 'rails', '~> 6.0'
+  gem "rspec-rails", '~> 5'
```
Comment (cysjonathan) [inline Appraisals:3]: [chore(dep): modify rspec version for each rails ver](https://github.com/Coursemology/activerecord-userstamp/pull/4/commits/5de7cc00f9fd24d505d97ada870600175b357248)
should squash with  
first commit [chore(test): setup test and change requirement](https://github.com/Coursemology/activerecord-userstamp/pull/4/commits/db72db7cc2ab9fb6172e59a17ea85bae98ac1944)

Clean these up?

**Review by cysjonathan — CHANGES_REQUESTED** [review]

File: README.md
Comment (bivanalhar) [reply README.md:1]: okay

File: Appraisals (hunk 1 above)
Comment (bivanalhar) [reply Appraisals:3]: done

**Review by cysjonathan — APPROVED** [review]

**Pull request merged**